@runningman84 it looks like [linting failed](https://circleci.com/gh/helm/charts/71456?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) due to trailing spaces in the `values.yml` file.  Can you please take a look?

```
stable/home-assistant/values.yaml
  154:30    error    trailing spaces  (trailing-spaces)

Error linting charts: Error processing charts
```

--- a/stable/home-assistant/values.yaml
+++ b/stable/home-assistant/values.yaml
@@ -151,7 +151,7 @@ configurator:
     loadBalancerSourceRanges: []
     # nodePort: 30000
 
-## Add support for Prometheus 
+## Add support for Prometheus
 # settings has to be enabled in configuration.yaml
 # https://www.home-assistant.io/components/prometheus/
 monitoring:

